source/extensions/transport_sockets/tls/context_impl.cc

```diff
@@ -335,6 +335,13 @@ ContextImpl::ContextImpl(Stats::Scope& scope, const Envoy::Ssl::ContextConfig& c
     }
   }
 
+  // As late as possible, run the custom SSL_CTX configuration callback on each
+  // SSL_CTX.
+  auto sslctx_cb = config.sslctx_cb();
+  for (TlsContext& ctx : tls_contexts_) {
```

Comment:
Could you wrap this in `if (sslctx_cb) { ... }` to only call if set? Calling empty lambdas seems wasteful.

Reply:
Agreed, done.